js/bybit.js

```diff
@@ -1218,15 +1223,28 @@ module.exports = class bybit extends Exchange {
         };
         const marketTypes = this.safeValue (this.options, 'marketTypes', {});
         const marketType = this.safeString (marketTypes, symbol);
-        let method = (marketType === 'linear') ? 'privateLinearGetOrderSearch' : 'v2PrivateGetOrder';
+        let method = undefined;
+        if (marketType === 'linear') {
+            method = 'privateLinearGetOrderSearch';
+        } else if (marketType === 'inverse') {
+            method = 'v2PrivateGetOrder';
+        } else if (marketType === 'inverseFuture') {
+            method = 'futuresPrivateGetOrder';
+        }
```

Comment:
Avoid the use of else if statement here.
```suggestion
        if (marketType === 'linear') {
            method = 'privateLinearGetOrderSearch';
        }
        if (marketType === 'inverse') {
            method = 'v2PrivateGetOrder';
        }
        if (marketType === 'inverseFuture') {
            method = 'futuresPrivateGetOrder';
        }
```

Reply:
fixed

Reply:
@kroitor done